README.md

```diff
@@ -14,7 +14,7 @@ For other versions of our browser, please see:
 
 To download the latest release, [see our releases page](https://github.com/brave/browser-laptop/releases).
 
-For a more user-friendly download page, [please visit our website](https://brave.com/downloads.html).
+You can also [visit our website](https://brave.com/downloads.html) to get the latest stable release (along with a more user-friendly download page).
```

Comment:
We also appear to support https://brave.com/downloads, without the .html extension. May be preferred, aesthetically.